src/libraries/System.Security.Cryptography.Algorithms/tests/RandomNumberGeneratorTests.cs

```diff
@@ -229,6 +229,35 @@ public static void GetBytes_InvalidArgs_Base()
             }
         }
 
+        [Fact]
+        public static void GetBytes_Int_Negative()
+        {
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("byteCount", () =>
+                RandomNumberGenerator.GetBytes(-1));
+        }
+
+        [Fact]
+        public static void GetBytes_Int_Empty()
+        {
+            byte[] result = RandomNumberGenerator.GetBytes(0);
+            Assert.Same(Array.Empty<byte>(), result);
+        }
+
+        [Fact]
+        public static void GetBytes_Int_RandomDistrobution()
```

Comment:
Nit for myself that I will fix pending an open question: small Distribution correctly.